### PR TITLE
[COS-38] added validation for file upload

### PR DESCRIPTION
**Pull request by victorjeman**
@lukeferguson I've made some changes in directive. Now we can specify the file max size, type that should be accepted.

### CODE REVIEW

File: src/core/javascripts/directives/file_upload.js.coffee [hunk 1]
```diff
@@ -22,6 +26,8 @@ angular.module('BB.Directives').directive 'bbFileUpload', () ->
   controller : 'FileUpload',
   templateUrl: 'file_upload.html'
   link : (scope, element, attrs) ->
+    scope.maxSize = attrs.maxSize
+    scope.accept = attrs.accept
```
Comment (lukeferguson) [inline src/core/javascripts/directives/file_upload.js.coffee]: Hey @victorjeman, you might want to consider using isolate scope which saves you from having read from the `attrs` yourself. Also protects the directive against scope pollute too.

File: src/core/javascripts/directives/file_upload.js.coffee [hunk 2]
```diff
@@ -42,6 +48,8 @@ angular.module('BB.Controllers').controller 'FileUpload', ($scope, Upload) ->
   * @param {array} errFiles errors array
   ###
   $scope.uploadFile = (item, file, errFiles, existing) ->
+    $scope.errFile = errFiles and errFiles[0]
```
Comment (lukeferguson) [inline src/core/javascripts/directives/file_upload.js.coffee]: All variables should be `snaked_cased`

Comment (victorjeman) [inline src/core/javascripts/directives/file_upload.js.coffee]: Ok, I will change them all.

**Comment by victorjeman** [comment]
@lukeferguson I've made the changes.

**Pull request closed without merging**